### DISCUSSION TITLE
Where can I view compose stack logs outside of the web UI?

### DISCUSSION THREAD

**Discussion by MulverineX** [louislam/dockge#310] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

Currently I am trying to diagnose a problem with one of my containers, and I would like to be able to view the compose log in a terminal, as the web UI terminal is pretty restrictive with no search function.

### 🐻 Dockge Version

1.3.3

### 💻 Operating System and Arch

Arch Linux x64

### 🌐 Browser

Ungoogled Chromium 120.0.6099.71

### 🐋 Docker Version

Docker 24.0.7

**Comment by louislam** [comment]
You can check it by `docker compose logs --help`

`docker compose logs <service name>`

**Reply by trythatagain** [reply]
I also like dozzle (https://github.com/amir20/dozzle) for viewing logs thru a web browser
